metrics-benchmarks/src/main/java/com/codahale/metrics/benchmarks/ReservoirBenchmark.java

```diff
@@ -60,6 +63,12 @@ public Object perfSlidingTimeWindowReservoir() {
         return slidingTime;
     }
 
+    @Benchmark
+    public Object perfLockFreeExponentiallyDecayingReservoir() {
+        lockFreeExponential.update(nextValue);
+        return slidingTime;
```

Comment:
This should return `lockFreeExponential` I think - all the other examples return the object under test

Reply:
Good catch, thanks
```suggestion
        return lockFreeExponential;
```